core/src/config.rs

```diff
@@ -46,7 +46,7 @@ struct Cli {
     #[clap(long)]
     pub chain_id: ChainId,
     /// The address of the darkpool contract, defaults to the Goerli deployment
-    #[clap(long, value_parser, default_value = "0x1e7857cdd3d73838b0e053be1fa068aa15113793fea95ab663501789d3d0b51")]
```

Comment:
I deployed a new version of the contract to goerli with an updated interface given the encryption redesign